lib/Doctrine/Annotations/Metadata/Exception/InvalidAnnotationTarget.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations\Metadata\Exception;
+
+use InvalidArgumentException;
+use function sprintf;
+
+final class InvalidAnnotationTarget extends InvalidArgumentException
+{
+    public static function fromInvalidBitmask(int $bitmask) : self
+    {
+        return new self(sprintf('Annotation target %d is not valid bitmask of allowed targets.', $bitmask));
```

Comment:
I would suggest putting the target into double quotes to better separate them from the rest of the exception message:
```suggestion
        return new self(sprintf('Annotation target "%d" is not valid bitmask of allowed targets.', $bitmask));
```